src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
+  let removePropertiesClone = {};
```

Comment:
```suggestion
  let propertiesClone = {...state};
```
this is clone only for remove? lets make it as a clone for current state